cmd/provision/provision.go

```diff
@@ -292,6 +292,9 @@ func (p *provision) run(printf shared.FormatFn) error {
 			if err != nil {
 				return errors.Wrapf(err, "creating analytics secret data")
 			}
+		} else {
+			shared.Errorf("\nWARNING: No analytics service account given via --analytics-sa.")
```

Comment:
I just realized that we're using `shared.Errorf()` directly in a few places. Although the net result is the same, we should actually be using the passed param: `printf shared.formatFn` - so, `printf.Errorf()`.

Reply:
I think printf it self "implements" shared.FormatFn, just like shared.Errorf(). Calling printf would emit to stdout. Also verbosef is not necessarily enabled.